packages/web/src/components/toggle-collapse-button/ToggleCollapseButton.tsx

```diff
@@ -0,0 +1,56 @@
+import { useState, useCallback } from 'react'
+
+import { ResizeObserver } from '@juggle/resize-observer'
+import cn from 'classnames'
+import useMeasure from 'react-use-measure'
+
+import { ReactComponent as IconCaretDownLine } from 'assets/img/iconCaretDownLine.svg'
+import { ReactComponent as IconCaretUpLine } from 'assets/img/iconCaretUpLine.svg'
+
+import styles from './ToggleCollapseButton.module.css'
+
+export const ToggleCollapseButton = ({
+  className,
+  toggleButtonClassName,
+  showByDefault = false,
+  collapsedHeight = 0,
+  showText,
+  hideText,
+  children
+}: {
+  className?: string
```

Comment:
also i personally like always breaking up the prop types into separate type, so `ToggleCollapseButtonProps`, just makes it easier to read and export when needed.